### PR TITLE
refs #15058 - enable mongo auth

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by ekohl** [comment]
ACK

File: manifests/init.pp [hunk 1]
```diff
@@ -64,6 +68,9 @@
   $enable_ostree = $katello::params::enable_ostree,
   $candlepin_event_queue = $katello_devel::params::candlepin_event_queue,
 
+  $pulp_db_username = $katello_devel::params::pulp_db_username,
+  $pulp_db_password = $katello_devel::params::pulp_db_password,
```
Comment (ehelms) [inline manifests/init.pp]: I don't see these defined anywhere in params.pp

Comment (ekohl) [inline manifests/init.pp]: They are inherited from katello::params once https://github.com/Katello/puppet-katello/pull/126 is merged.

Comment (ehelms) [inline manifests/init.pp]: How? This says `katello_devel::params`

Comment (ekohl) [inline manifests/init.pp]: Which in turn inherits from ```katello::params``` (https://github.com/Katello/puppet-katello_devel/blob/4d38cf0cad1515bf29fd9b5e0047094404082066/manifests/params.pp#L2). Ugly, but it works ;)

Comment (ehelms) [inline manifests/init.pp]: Touche :)

Comment (stbenjam) [inline manifests/init.pp]: I can explicitly add them here if you want, not a big fan of this many layers deep inheritance to figure out where the variable comes from.

Comment (ekohl) [inline manifests/init.pp]: I think we have the same pattern for some other parameters as well so I'd either add them all (and remove the inheritance) or leave it as it is now.

**Comment by ekohl** [comment]
ACK

**Pull request merged**